backend/src/Installer.ts

```diff
@@ -64,25 +63,26 @@ export default class Installer {
    * to decide if we install connectd or not on startup.
    */
   isInstalled() {
-    if (binaryInstaller.inProgress) return false
-    const check = this.dependencyNames.concat(this.binaryName)
-    const missing = check.find(fileName => !this.fileExists(fileName))
-    d('IS INSTALLED?', { installed: !missing })
-    return !missing
+    return true
+    // if (binaryInstaller.inProgress) return false
+    // const check = this.dependencyNames.concat(this.binaryName)
+    // const missing = check.find(fileName => !this.fileExists(fileName))
+    // d('IS INSTALLED?', { installed: !missing })
+    // return !missing
```

Comment:
We should clean up this commented out stuff - just remove the stuff that isn't used any more.